cf_xarray/tests/test_accessor.py

```diff
@@ -429,6 +429,27 @@ def test_bounds():
     assert_identical(actual, expected)
 
 
+def test_bounds_to_corners():
```

Comment:
this is not testing the `mollwds` dataset that you created?

Reply:
Ach saperlipopette, I forgot to push the new `test_helpers.py` file.